charts/karma/README.md

```diff
@@ -100,3 +100,29 @@ existingSecretConfig:
 ```
 
 **NOTE:** you can either use `existingSecretConfig` or `configMap`, it cannot be both.
+
+
+### kthxbye sidecar
+kthxbye is a tiny daemon that can help with managing short lived acknowledged silences.
+It will continuously extend short lived acknowledgement silences if there are alerts firing against those silences,
+which means that the user doesn't need to worry about setting proper duration for such silences.
+
+```yml
```

Comment:
Probably not needed to reproduce whole content of values.yaml here.

Reply:
shortened the values file in readme